### PR TITLE
58645: Remove grunt-jsvalidate from build process

### PR DESCRIPTION
**Pull request by kadamwhite**
Trac ticket: https://core.trac.wordpress.org/ticket/58645

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -1036,29 +1036,6 @@ module.exports = function(grunt) {
 				}
 			}
 		},
-		jsvalidate:{
```
Comment (joemcgill) [inline Gruntfile.js:1039]: There are a few other places in the Gruntfile that references jsvalidate. We should remove all of these instances as well.

**Review by joemcgill — CHANGES_REQUESTED** [review]

File: Gruntfile.js (hunk 1 above)
Comment (kadamwhite) [reply Gruntfile.js:1039]: Done

File: Gruntfile.js [hunk 2]
```diff
@@ -1819,7 +1818,7 @@ module.exports = function(grunt) {
 			if ( action !== 'deleted' ) {
 				grunt.config( [ 'copy', 'dynamic-js', 'files' ], files );
 			}
-		// For the webpack builds configure the jsvalidate task to only check those files build by webpack.
+		// For the webpack builds configure the task to only check those files built by webpack.
```
Comment (joemcgill) [inline Gruntfile.js:1821]: I'm curious now that we're removing jsvalidate if this whole `else if` could be removed or if it's still useful for some reason. I don't see that as a blocker to this patch though, so going to pre-approve.

**Review by joemcgill — APPROVED** [review]

File: Gruntfile.js (hunk 2 above)
Comment (kadamwhite) [reply Gruntfile.js:1821]: I wasn't certain so erred on the side of not changing the block, since it affects a non-validate-specific task.

**Review by westonruter — APPROVED** [review]
Same changes as I did in https://github.com/WordPress/wordpress-develop/pull/4562/commits/0752236a55a5b7c67574f5e5ce86db27d58fd9be (for #4562) to allow builds to pass when utilizing ES6+ syntax.

**Comment by khansahan9864** [comment]
was this was successful


{{{
#!html
<a href="https://7met.com"></a>
<a href="https://baddiehub.ca"></a>
<a href="https://sarkarijob.com"></a>
<a href="https://comtex-nj.com"></a>
<a href="https://panthercontractingllcnj.com"></a>
}}}

**Comment by kadamwhite** [comment]
Committed in https://core.trac.wordpress.org/changeset/56247

**Pull request closed without merging**